evalml/automl/automl_search.py

```diff
@@ -645,7 +641,13 @@ def _compute_cv_scores(self, pipeline, X, y):
         start = time.time()
         cv_data = []
         logger.info("\tStarting cross validation")
-        for i, (train, test) in enumerate(self.data_split.split(X, y)):
+        X = _convert_to_woodwork_structure(X)
+        y = _convert_to_woodwork_structure(y)
```

Comment:
In case `add_to_rankings` is called (not from `search`). From search, this should basically be a no-op, as the conversion to woodwork already happened.